state-chain/pallets/cf-environment/src/lib.rs

```diff
@@ -379,7 +381,7 @@ pub mod pallet {
 		) -> DispatchResult {
 			T::EnsureGovernance::ensure_origin(origin)?;
 
-			ensure!(params.are_valid(), DispatchError::Other("Invalid parameters"));
+			ensure!(params.are_valid(), Error::<T>::InvalidConsolidationParameters);
```

Comment:
The changes like this make sense where we're actually changing the type, but I don't really understand the above change (and the similar ones where we're still using DispatchError::Other).

Reply:
Yeah because if we just use DispatchError::Other, you will get an error with no information. Here with a pallet error, you can get information about the error type from the name